### PR TITLE
rbac: add user-facing clusterroles

### LINKED ISSUE #584: rbac: user facing clusterroles
After postgres-operator has been installed on a cluster, non-clusteradmin users of the k8s cluster cannot view or create databases by default.

```
kubectl get postgresqls
Error from server (Forbidden): postgresqls.acid.zalan.do is forbidden: User "Erik.Inge.Bolso@nrk.no" cannot list resource "postgresqls" in API group "acid.zalan.do" in the namespace "default"

```
Should probably have some example clusterroles in the manifests directory that can be added to give users of the k8s access to view and make postgreses. I'll make a pull request.

### PR DESCRIPTION
**Pull request by knan-nrk**
Fixes #584. Adds user facing clusterroles so kubectl get postgresql and kubectl create postgresql works for view and edit/admin default rolebindings.

### CODE REVIEW

File: manifests/user-facing-clusterroles.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  labels:
+    rbac.authorization.k8s.io/aggregate-to-admin: "true"
+    rbac.authorization.k8s.io/aggregate-to-edit: "true"
+  name: zalando-postgres-operator:users:edit
+rules:
+- apiGroups:
+  - acid.zalan.do
+  resources:
+  - postgresqls
+  - postgresqls/status
```
Comment (FxKu) [inline manifests/user-facing-clusterroles.yaml]: `OperatorConfiguration` CRD is missing under `resources`. Any reason?

**Comment by FxKu** [comment]
If we add these cluster roles. We need to document them somewhere. [User](https://github.com/zalando/postgres-operator/blob/master/docs/user.md) docs probably makes sense, as these roles are meant for users :)

File: manifests/user-facing-clusterroles.yaml (hunk 1 above)
Comment (knan-nrk) [reply manifests/user-facing-clusterroles.yaml]: Hmm. This is user-facing. I don't think users of the k8s cluster that want to create a database need to see the operator config in order to create a database, I consider that private to the operator and appropriate to stay at clusteradmin level permission to see.

**Comment by knan-nrk** [comment]
Updated with a little bit of doc and went from two to three roles.

File: manifests/user-facing-clusterroles.yaml (hunk 1 above)
Comment (FxKu) [reply manifests/user-facing-clusterroles.yaml]: I thought about that, too. Fine for me.
I think, `postgresqls/status` could also be removed from the edit role. For admin and view it makes sense.

Comment (knan-nrk) [reply manifests/user-facing-clusterroles.yaml]: Agreed. Updated. :)

**Comment by FxKu** [comment]
:+1:

**Comment by sdudoladov** [comment]
:+1:

**Comment by sdudoladov** [comment]
@knan-nrk  thank you for yet another contribution :+1:

**Pull request merged**